lib/fabrik/DirectorManager.js

```diff
@@ -223,6 +228,34 @@ class DirectorManager extends BaseManager {
       });
   }
 
+  executeActions(phase, deploymentName, params, args) {
+    return Promise.try(() => {
+      if (this.service.actions || this.plan.actions) {
+        const cumilativeActions = this.service.actions ? (this.plan.actions ? `${this.service.actions},${this.plan.actions}` : this.service.actions) :
+          this.plan.actions;
+        logger.info(`actionsToPerform - @service - ${this.service.actions} , @plan - ${this.plan.actions}`);
```

Comment:
For most of the cases this would be empty. Better to move this logger to line #246. Can avoid unnecessary logs.